dm/pkg/parser/common.go

```diff
@@ -318,6 +318,7 @@ func SplitDDL(stmt ast.StmtNode, schema string) (sqls []string, err error) {
 		v.Table = table
 
 		return sqls, nil
+	case *ast.CreateViewStmt:
```

Comment:
How about add all `will be skipped in SkipQueryEvent()` ddl here? @lance6716 

https://github.com/pingcap/tiflow/blob/95ddc8a60b9022320c7a97e0e71d3db5763a59df/dm/pkg/utils/util.go#L45-L100

Reply:
or we execute 

```go
	if utils.IsBuildInSkipDDL(qec.originSQL) {
		return true, nil
	}
```

before `SpiltDDL`

Reply:
there're more cases than CREATE VIEW. I think we should do as https://github.com/pingcap/tiflow/blob/95ddc8a60b9022320c7a97e0e71d3db5763a59df/dm/syncer/syncer.go#L2366-L2380